test/client/app/components/higherOrder/hasConfig.test.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { expect } from 'chai';
+import { shallow } from 'enzyme';
+import hasConfig from '../../../../../assets/app/components/higherOrder/hasConfig';
+
+const mock = React.createClass({
+  render() {
+    return <div></div>
+  }
+});
+
+describe.only('hasConfig()', () => {
```

Comment:
we should remove the "only" here as it causes us to run only these two client tests.